### PR TITLE
solution

### PR DESCRIPTION
**Pull request by lukacsja**
[DEMO LINK](https://lukacsja.github.io/react_movies-list/)

### CODE REVIEW

File: src/components/MovieCard/MovieCard.tsx [hunk 1]
```diff
@@ -1,7 +1,55 @@
...
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
```
Comment (Grayfox147) [inline src/components/MovieCard/MovieCard.tsx:12]: I would recommend you to destructure the movie, so you dont need to use movie.imgUrl but only imgUrl. It makes the code to look cleaner and more organize.

**Review by Grayfox147 — COMMENTED** [review]
Overall really good implementation! I believe is a great work!

File: src/components/MovieCard/MovieCard.tsx (hunk 1 above)
Comment (naziamoff) [reply src/components/MovieCard/MovieCard.tsx:12]: agree with @Grayfox147

**Review by naziamoff — APPROVED** [review]
Well done

**Review by naziamoff — CHANGES_REQUESTED** [review]
oops, demo seems to be broken

**Review by naziamoff — APPROVED** [review]
🔥

**Pull request closed without merging**